### ISSUE TITLE
PowerShell module

### ISSUE THREAD

**Issue by vmsilvamolina** [sendgrid/hacktoberfest#2]
I think that it would a good idea develop a module for PowerShell

**Comment by movinalot** [comment]
Until then...

Invoke-RestMethod -Uri https://api.sendgrid.com/v3/mail/send -Method Post -Headers @{"Authorization" = "Bearer $SENDGRID_API_KEY"; "Content-Type" = "application/json"} -Body '{"personalizations": [{"to": [{"email": "test@example.com"}]}],"from": {"email": "test@example.com"},"subject": "Sending with SendGrid is Fun","content": [{"type": "text/plain", "value": "and easy to do anywhere, even with PowerShell"}]}'

**Comment by vmsilvamolina** [comment]
@movinalot Yes, I know that method, but my idea is create something more elaborate: If you have a module, you can install it, and use cmdlet with a friendly-name-functions linke "Get-SendGridConfiguration". Obviusly, for build the module you can use that API.

**Comment by thinkingserious** [comment]
Great idea @vmsilvamolina! Would that be something that would be deployed to Nuget?

@movinalot would you mind making a [PR against our docs](https://github.com/sendgrid/docs) with your example?

**Comment by vmsilvamolina** [comment]
@thinkingserious I think it fits better on the PowerShell library https://www.powershellgallery.com/. You can install the module using the cmdlet `Install-Module <moduleName>`.

About the examples... can I help with the PR over the docs?

**Comment by vmsilvamolina** [comment]
@aroach how can I continue with this?, You have a module with a function, what is the next step? Does I work on your repo or I make a new pull request for the SendGrid organization?

**Comment by aroach** [comment]
@vmsilvamolina since it's a new language, I think it warrants a new repo in our Org, yeah?  @thinkingserious does that seem OK to you?

**Comment by aroach** [comment]
@vmsilvamolina in the meantime, you're welcome to fork it and move it forward.  We can transfer it to the SendGrid org when the time is right.

**Comment by maor29** [comment]
Hi,

I need to create a powershell script to create the API Keys in SendGrid. Does anyone know how to do that?

**Comment by aroach** [comment]
@maor29 have a look at this project. You should just need to tweak this file to agree with the API Keys API 

https://github.com/aroach/sendgrid-powershell/blob/4918a8cce905a8489ada25a6efb3da4dd51af216/sendgrid.powershell.psm1#L71
